src/rendering/react/components/CronJob/CronJobDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import { FC } from "react";
+import { MittwaldAPIV2 } from "@mittwald/api-client";
+import { SingleResult } from "../SingleResult.js";
+import { Value } from "../Value.js";
+import { Box, Text } from "ink";
+import { IDAndShortID } from "../IDAndShortID.js";
+import { CreatedAt } from "../CreatedAt.js";
+import { useProject } from "../../../../lib/project/hooks.js";
+import { useAppInstallation } from "../../../../lib/app/hooks.js";
+import { FormattedDate } from "../FormattedDate.js";
+import CronjobCronjob = MittwaldAPIV2.Components.Schemas.CronjobCronjob;
+import CronjobCronjobUrl = MittwaldAPIV2.Components.Schemas.CronjobCronjobUrl;
+import CronjobCronjobCommand = MittwaldAPIV2.Components.Schemas.CronjobCronjobCommand;
+
+type CronJobComponent = FC<{ cronjob: CronjobCronjob }>;
+
+const CronJobNextExecution: CronJobComponent = ({ cronjob }) => {
+  if (!cronjob.nextExecutionTime) {
+    return <Value notSet />;
+  }
+
+  return (
+    <Value>
+      <FormattedDate relative date={cronjob.nextExecutionTime} />
+    </Value>
+  );
+};
+
+const CronJobExecutionTargetURL: FC<{ dest: CronjobCronjobUrl }> = ({
+  dest,
+}) => {
+  return (
+    <SingleResult
+      title="EXECUTION TARGET"
+      rows={{
+        URL: <Value>{dest.url}</Value>,
+      }}
+    />
+  );
+};
+
+const CronJobExecutionTargetCommand: FC<{ dest: CronjobCronjobCommand }> = ({
```

Comment:
`dest`? wtf is `dest`? How do you get from `CronjobCronjobCommand` to `dest`? Can we name it `command` or `cmd` or something similar?

Reply:
In the API spec, a cronjob has a "destination", which can be either an URL or a command -- which is where this naming comes from. I've renamed it to `command`, for this component.